javascript/answers/anamika-katyalq22.html

```diff
@@ -0,0 +1,67 @@
+<html>
+    <head>
+
+        <!-- Global site tag (gtag.js) - Google Analytics -->
+        <script async src="https://www.googletagmanager.com/gtag/js?id=UA-157837769-1"></script>
+        <script>
+        window.dataLayer = window.dataLayer || [];
+        function gtag(){dataLayer.push(arguments);}
+        gtag('js', new Date());
+        gtag('config', 'UA-157837769-1');
+        </script>
+
+        <title>
+            JavaScript | InterQues | Coding Ninjas
+        </title>
+
+        <link rel="shortcut icon" type="image/png" href="../../assets/favicon.png"/>
+
+        <meta name="description" content="What is the output of the code below ?">
+    </head>
+    <body>
+
+        <div>
+            <img src="../../assets/CN_logo.png" height='72px' alt="Coding Ninjas Logo">
+        </div>
+
+        <div>
+            <p>
+                <small>
+
+                    <a href="../../index.html">Home</a>
+                    &gt; <a href="../index.html">JavaScript Questions</a>
+                    &gt; What is the output of the code below ?
+
+                </small>
+
+            </p>
+        </div>
+
+
+        <div>
+            <div>
+                <h4>
+                   What is the output of the code below ?
+                </h4>
+            </div>
+            <div>
+                <strong>Answer:</strong>
+                <p>Error 1, Success4</p>
```

Comment:
@Anamika2716 please try to give an explanation for the output.

Reply:
Added explanation

Reply:
It does not show any explanation added. 😕 
Check if your changes were committed or not.